x-pack/test/functional/services/ml/common_api.ts

```diff
@@ -9,16 +9,22 @@ import { ProvidedType } from '@kbn/test';
 
 import { FtrProviderContext } from '../../ftr_provider_context';
 
-export const COMMON_REQUEST_HEADERS = {
+const COMMON_REQUEST_HEADERS = {
   'kbn-xsrf': 'some-xsrf-token',
 };
 
 export type MlCommonAPI = ProvidedType<typeof MachineLearningCommonAPIProvider>;
 
 export function MachineLearningCommonAPIProvider({}: FtrProviderContext) {
   return {
-    async getCommonRequestHeader() {
-      return COMMON_REQUEST_HEADERS;
-    },
+    getCommonRequestHeader,
   };
 }
+
+export function getCommonRequestHeader(apiVersion?: string) {
```

Comment:
This is currently called like this:
```
getCommonRequestHeader('1')
```
which is nice and short. I'm just wondering if it can be harder to read in case you don't know what `'1'` is. In that scenario, something like
```
getCommonRequestHeader({apiVersion: '1'})
```
would be easier to consume. But on the other hand, it adds more overhead to the helper method calls and most folks are using IDEs with parameter hints anyway, so it might not be an issue at all. What do you think?

Reply:
Fair point, I'm happy to leave it as is.